src/main/java/com/phonebookservice/server/FileAdapter.java

```diff
@@ -0,0 +1,12 @@
+package main.java.com.phonebookservice.server;
+
+import main.java.com.phonebookservice.model.Contact;
+
+public class FileAdapter implements DBIf {
+
+    @Override
+    public void create(final Contact contact) {
+
```

Comment:
If you aren't going to implement this method in this code review, you should throw `UnsupportedOperationException`

https://docs.oracle.com/javase/7/docs/api/java/lang/UnsupportedOperationException.html

Also write a test case validate this method throws UnsupportedOperationException